tests/test_schema.py

```diff
@@ -254,6 +256,17 @@ def test_double_quote(self):
         assert rows == [{"column_1": Decimal("6.66")}]
         assert not errors
 
+    def test_asymetrical_quote(self):
+        schema = Schema.build(
+            {
+                "file_type": "csv",
+                "has_header": True,
+                "fields": [{"key": "column_1", "type": "str", "column-number": 1}],
+            }
+        )
+        rows, errors = schema.parse(b'column_1\n"test ""quoting""')
+        assert rows == [{"column_1": '"test ""quoting""'}]
```

Comment:
Ah je pensais que ça ferait 'test quoting' !

Reply:
ça marche toujours pour la partie portugal du coup? Ou tu veux test ça côté ext-is direct?

Reply:
ah bah non, ça ne te le fera jamais à ta place

Reply:
pour la partie portugal, faut que je mette à jour mes tests côté ext mais ça casse rien